### ISSUE TITLE
Dropdowns - Select, Multiselect components have accessibility violations

### ISSUE THREAD

**Issue by tepahk** [zendeskgarden/react-components#1376]
## Expectations
- Dropdowns > **Select**
   - Component passes all accessibility tests 
- Dropdowns > **Multiselect**
   - Component passes all accessibility tests 

## Reality
- Dropdowns > **Select**
   - Component has accessibility violations:
      - `Ensure interactive controls are not nested`
- Dropdowns > **Multiselect**
   - Component has accessibility violations:
      - `Elements must only use allowed ARIA attributes`
      - `Required ARIA attributes must be provided`
      - `Certain ARIA roles must be contained by particular parents`

## Steps to Reproduce

1. Visit https://zendeskgarden.github.io/react-components/?path=/story/packages-dropdowns-select--select
1. View Accessibilty tab in Addons panel
1. Observe violations
1. Visit https://zendeskgarden.github.io/react-components/?path=/story/packages-dropdowns-multiselect--multiselect
1. View Accessibilty tab in Addons panel
1. Observe violations

### Fine Print

- Component: Select, Multiselect
- Version: 8.53.0
- Browsers: all

_This is not an exhaustive list of all violations, just a few we observed while extending Garden._

**Comment by jzempel** [comment]
@tepahk thanks for raising this issue. The Garden team is aware of the dropdown accessibility issues and has them on our radar as part of a larger component a11y sweep. I don't have a definite target timeframe for a fix. Stay tuned – we'll get back to you as plans materialize.

**Comment by jessicachou** [comment]
@jzempel this issue came up again recently in some web conversations, so do you have an update on the status or ETA for when this might get fixed? thanks!!

**Comment by jzempel** [comment]
@tepahk the refactored `Combobox` is now available :tada:. Please see the following references for details:
- https://www.npmjs.com/package/@zendeskgarden/react-dropdowns.next
- https://zendeskgarden.github.io/react-components/?path=/story/packages-dropdowns-next-combobox--uncontrolled
- https://codesandbox.io/s/combobox-demo-vhhlgh?file=/src/ComboboxNext.tsx
